src/run.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+// src/run.php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+$setAutoloader();
+$terminus = new Terminus();
+$command_factory = new AnnotatedCommandFactory();
+//$formatter_manager = new FormatterManager();
+
+$discovery = new CommandFileDiscovery();
+$command_files = $discovery->discover(__DIR__, '\Commands');
+foreach ($command_files as $command_class) {
+  $command_instance = new $command_class();
+  //$command_factory->commandProcessor()->setFormatterManager($formatter_manager);
+  $command_list = $command_factory->createCommandsFromClass($command_instance);
+  foreach ($command_list as $command) {
+    $terminus->add($command);
+  }
+}
+
+/**
+  * Sets the autoloader up to retrieve files as necessary
+  *
+  * @return void
+  */
+function setAutoloader() {
```

Comment:
I'm not sure I understand how your point is different than my suggestion. You want people pulling the `master` branch to get the alpha 1.x work, and people using the phar or installing via composer ~0 to get the old version?  That still works.

It doesn't seem worthwhile to try to maintain backwards compatibility just to get more alpha testers, or to try to run a version of Terminus with both old and new commands. I think that would slow you down a lot for little real gain.  My opinion, YMMV.